prettier/src/index.ts

```diff
@@ -174,13 +175,19 @@ const preserveFormatting = (input) => {
   return input;
 };
 
+let firstFile = true;
+
 const parsers = {
   hubl: {
     astFormat: "hubl-ast",
     parse,
     locStart,
     locEnd,
     preprocess: (text) => {
+      if (firstFile) {
+        trackUsage();
+        firstFile = false;
+      }
```

Comment:
This code is executed for each file so this prevents `trackUsage` from being called multiple times